modules/social_features/social_group/src/Form/SocialGroupSettings.php

```diff
@@ -64,6 +128,34 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#weight' => 20,
     ];
 
+    // Cross-posting settings.
+    $form['cross_posting'] = [
+      '#type' => 'details',
+      '#title' => $this->t('Cross-posting settings'),
+      '#open' => FALSE,
+      '#weight' => 30,
+      '#tree' => TRUE,
+    ];
+
+    $form['cross_posting']['status'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enable cross-posting'),
+      '#description' => $this->t('If enabled, one node can be added as a group content to multiple groups.'),
+      '#default_value' => $config->get('cross_posting.status'),
+    ];
+
+    $form['cross_posting']['entity_types'] = [
+      '#type' => 'checkboxes',
+      '#title' => $this->t('Entity types'),
```

Comment:
@nechai I think it would be clearer to rename this label and other places in code it to `Content types` since we implement cross-posting only for content types, but not for other entities Posts.